cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/power/HostActivationExample.java

```diff
@@ -218,11 +218,19 @@ private Host createHost() {
 
         //Indicates if the Host will be powered on or not after creation
         final boolean activateHost = false;
+        
+        //Indicates Host Power On Delay
+        final double hostStartUpDelay = 5;
+        
+        //Indicates Host Power off Delay
+        final double hostShutDownDelay = 5;
```

Comment:
I suppose usually, the shutdown delay is smaller than the startup delay.
This way, some arbitrary smaller value will be better for the example.